### PR TITLE
Update Images Autodocs Workflow

### PR DESCRIPTION
**Pull request by erikaheidi**
This is the updated workflow to the latest version of Autodocs.

There are things to improve (like using rsync instead of copying the files), but I'd like to get this back ASAP so our tags history pages are always up-to-date with the latest image builds. 

TODO: the image specs page is currently not being built, because it will require a lot more work to switch from reading the YAML config files to query metadatada from chainctl.

Tested in [my EDU fork](https://github.com/erikaheidi/academy/pull/5) and it worked great!

### CODE REVIEW

File: .github/workflows/autodocs-images.yaml [hunk 1]
```diff
@@ -79,7 +92,6 @@ jobs:
           commit-message: Update Images Reference
           title: "[AutoDocs] Update Images Reference Docs"
           body: "${{ env.CHANGELOG }}"
-          signoff: true
```
Comment (jamonation) [inline .github/workflows/autodocs-images.yaml:82]: Does the signing happen implicitly with GitHub's key, or should this stay in to keep the Enforce integration happy?

File: .github/workflows/autodocs-images.yaml [hunk 2]
```diff
@@ -17,40 +17,53 @@ jobs:
...
         run: |
           mkdir -p "${{ github.workspace }}/${{ env.WORKDIR }}" && \
           chmod 777 "${{ github.workspace }}/${{ env.WORKDIR }}"
 
-      - name: "Copy current changelog to workdir"
+      - name: Set up Chainctl
+        uses: chainguard-dev/actions/setup-chainctl@main
+        with:
+          # This identity allows us to list the public Chainguard images.
+          identity: 720909c9f5279097d847ad02a2f24ba8f59de36a/a033a6fabe0bfa0d
```
Comment (jamonation) [inline .github/workflows/autodocs-images.yaml:43]: Should this be an environment variable in the repo, or a secret?

File: .github/workflows/autodocs-images.yaml [hunk 3]
```diff
@@ -17,40 +17,53 @@ jobs:
...
+
+      - name: Fetch images list
+        run: |
+          chainctl img ls --group 720909c9f5279097d847ad02a2f24ba8f59de36a -ojson > "${{ github.workspace }}/images-tags.json"
```
Comment (jamonation) [inline .github/workflows/autodocs-images.yaml:47]: Should this be an environment variable in the repo, or a secret?

**Review by jamonation — APPROVED** [review]
All looks good to me. I left a couple questions, but if you've already thought about them then feel free to merge away!

File: .github/workflows/autodocs-images.yaml (hunk 2 above)
Comment (ltagliaferri) [reply .github/workflows/autodocs-images.yaml:43]: +1 variable

File: .github/workflows/autodocs-images.yaml (hunk 3 above)
Comment (ltagliaferri) [reply .github/workflows/autodocs-images.yaml:47]: +1

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
If we can do the variables that would be better, thanks!

File: .github/workflows/autodocs-images.yaml (hunk 1 above)
Comment (erikaheidi) [reply .github/workflows/autodocs-images.yaml:82]: This was breaking the workflow: https://github.com/erikaheidi/academy/actions/runs/5580884160

The error said:
```

main
Unexpected error: error: gpg failed to sign the data:
[GNUPG:] BEGIN_SIGNING
failed to create rekor client: unable to fetch rekor public key from rekor: Get "https://rekor.sigstore.dev/api/v1/log/publicKey": GET https://rekor.sigstore.dev/api/v1/log/publicKey giving up after 1 attempt(s): context deadline exceeded
Error: failed to create rekor client: unable to fetch rekor public key from rekor: Get "https://rekor.sigstore.dev/api/v1/log/publicKey": GET https://rekor.sigstore.dev/api/v1/log/publicKey giving up after 1 attempt(s): context deadline exceeded
```
I can have another look and try to figure it out, I think something has probably changed with gitsign recently? Do you know @jamonation ?

Comment (erikaheidi) [reply .github/workflows/autodocs-images.yaml:82]: I think this was an intermittent issue, I just run it again in my forked repo and it worked.

**Pull request closed without merging**